src/dpdb/dpdb.js

```diff
@@ -19,7 +19,8 @@ var DPDB_CACHE = require('./dpdb-cache.js');
 var Util = require('../util.js');
 
 // Online DPDB URL.
-var ONLINE_DPDB_URL = 'https://storage.googleapis.com/cardboard-dpdb/dpdb.json';
+var ONLINE_DPDB_URL =
+  'https://rawgit.com/thoragio/webvr-polyfill-dpdb/master/dpdb.json';
```

Comment:
would you be able to publish the `master` branch of https://github.com/thoragio/webvr-polyfill-dpdb to GitHub Pages so it's served from https://thoragio.github.io/webvr-polyfill-dpdb? (the benefit is the caching is better on GitHub Pages than on RawGit.)

not absolutely necessary for this pass, but if you can easily do it, lmk. otherwise, we can probably still merge this (per @toji's approval).